src/main/kotlin/no/nav/syfo/personstatus/domain/PersonOversiktStatus.kt

```diff
@@ -30,6 +30,11 @@ data class PersonOversiktStatus(
         null, false, null, null,
         null, null, null, null, null, null,
     )
+    constructor(fnr: String, veilederIdent: String?) : this(
+        veilederIdent = veilederIdent, fnr = fnr, null, null, null,
+        null, false, null, null,
+        null, null, null, null, null, null,
+    )
```

Comment:
Er denne konstruktøren laget kun for å brukes i test? I så fall kan du vurdere å gjøre `val statusWithVeileder = PersonOversiktStatus(fnr).copy(veilederident)`

Reply:
Ja, det kan jeg jo! 🧠 Smart